mitmproxy/tools/web/app.py

```diff
@@ -654,6 +656,43 @@ def get(self):
         self.write(State.get_json(self.master))
 
 
+class ProcessList(RequestHandler):
+    @staticmethod
+    def process_to_dict(process):
+        return {
+            "is_visible": process.is_visible,
+            "executable": process.executable,
+            "is_system": process.is_system,
+            "display_name": process.display_name,
+        }
+
+    @staticmethod
+    def get_json():
+        processes = mitmproxy_rs.active_executables()
+        processes_json = [ProcessList.process_to_dict(process) for process in processes]
+        return {"processes": processes_json}
```

Comment:
Let's keep this simpler:
```suggestion
    @staticmethod
    def get_json():
        processes = mitmproxy_rs.active_executables()
        return [
            {
                "is_visible": process.is_visible,
                "executable": process.executable,
                "is_system": process.is_system,
                "display_name": process.display_name,
            }
            for process in processes
        ]
```